lib/app_web/live/stats_live.html.heex

```diff
@@ -0,0 +1,46 @@
+<main class="font-sans container mx-auto">
+  <div class="relative overflow-x-auto mt-12">
+    <h1 class="mb-12 text-xl font-extrabold leading-none tracking-tight text-gray-900 md:text-5xl lg:text-6xl dark:text-white">
+      User metrics
```

Comment:
We are very deliberately avoiding the word "User" in our projects; https://github.com/dwyl/app/issues/33

Reply:
removed. ✂️